qiita_pet/templates/sitebase.html

```diff
@@ -241,6 +243,11 @@
                 <li>
                   <a href="http://github.com/biocore/qiita">GitHub</a>
                 </li>
+                <li role="separator" class="divider"></li>
+                <li>
+                  <a href="#"><h6>{{qiita_version}}</h6></a>
```

Comment:
Hahaha, if it looks horrible then maybe two lines.

On (Feb-08-16|13:17), Antonio Gonzalez wrote:
> > @@ -241,6 +243,11 @@
> >                  <li>
> >                    <a href="http://github.com/biocore/qiita">GitHub</a>
> >                  </li>
> > +                <li role="separator" class="divider"></li>
> > +                <li>
> > +                  <a href="#"><h6>{{qiita_version}}</h6></a>
> 
> I tried adding in 1 line but it looks horrible. I can try in 2, what do you think? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1640/files#r52218901